app/models/address.rb

```diff
@@ -1,10 +1,10 @@
 class Address < ApplicationRecord
   # before_save :reformat_phone
 
-  belongs_to :location
+  belongs_to :location, required: false
```

Comment:
when you need to add one for a nested form. it's similar to how the item_location was done.